apps/src/lib/cli.rs

```diff
@@ -1201,21 +1211,130 @@ pub mod cmds {
                 .add_args::<args::InitGenesisValidator>()
         }
     }
+
+    /// Used as sub-commands (`SubCmd` instance) in `anoma` binary.
+    #[derive(Clone, Debug)]
+    pub enum EthBridgePool {
+        /// Construct a proof that a set of transfers is in the pool.
+        /// This can be used to relay transfers across the
+        /// bridge to Ethereum.
+        ConstructProof(args::BridgePoolProof),
+        /// Query the contents of the pool.
+        QueryPool(args::Query),
+    }
+
+    impl Cmd for EthBridgePool {
+        fn add_sub(app: App) -> App {
+            app.subcommand(ConstructProof::def().display_order(1))
+                .subcommand(QueryEthBridgePool::def().display_order(1))
+        }
+
+        fn parse(matches: &ArgMatches) -> Option<Self> {
+            let construct_proof = ConstructProof::parse(matches)
+                .map(|proof| Self::ConstructProof(proof.0));
+            let query_pool = QueryEthBridgePool::parse(matches)
+                .map(|q| Self::QueryPool(q.0));
+            construct_proof.or(query_pool)
+        }
+    }
+
+    impl SubCmd for EthBridgePool {
+        const CMD: &'static str = BRIDGE_POOL_CMD;
+
+        fn parse(matches: &ArgMatches) -> Option<Self> {
+            Cmd::parse(matches)
+        }
+
+        fn def() -> App {
+            App::new(Self::CMD)
+                .about(
+                    "Functionality for interacting with the Ethereum bridge \
+                     pool. This pool holds transfers waiting to be relayed to \
+                     Ethereum.",
+                )
+                .subcommand(ConstructProof::def().display_order(1))
+                .subcommand(QueryEthBridgePool::def().display_order(1))
+        }
+    }
+
+    #[derive(Clone, Debug)]
+    pub struct AddToEthBridgePool(pub args::EthereumBridgePool);
+
+    impl SubCmd for AddToEthBridgePool {
+        const CMD: &'static str = "add-transfer";
```

Comment:
Would call this something like `add-erc20-transfer` or otherwise put it under some subcommand of `namadac` - otherwise it is just `namadac add-transfer`. This command isn't showing in the help string for some reason when you run `namadac`

Reply:
I renamed the command. It shows up for me in the help menu.